config/slist.c

```diff
@@ -218,6 +218,99 @@ static intptr_t slist_native_get(const struct ConfigSet *cs, void *var,
   return (intptr_t) list;
 }
 
+/**
+ * number_string_plus_equals - Add to a Slist by string - Implements ConfigSetType::string_plus_equals()
+ */
+static int slist_string_plus_equals(const struct ConfigSet *cs, void *var,
+                                     const struct ConfigDef *cdef,
+                                     const char *value, struct Buffer *err)
+{
+  if (!cs || !cdef)
+    return CSR_ERR_CODE; /* LCOV_EXCL_LINE */
+
+  int rc = CSR_SUCCESS;
+
+  /* Store empty strings as NULL */
+  if (value && (value[0] == '\0'))
+    return rc |= CSR_SUC_NO_CHANGE; // return no change
+
+  struct Slist *list = *(struct Slist **) var;
+  struct Slist *orig = slist_dup(list);
```

Comment:
The naming of these two seems a bit odd.
I think I'd go for `orig = *var` and `copy = dup(orig)`